src/fusor/models.py

```diff
@@ -125,85 +120,71 @@ class TranscriptSegmentElement(BaseStructuralElement):
+        exon_start = values.get("exonStart")
+        exon_end = values.get("exonEnd")
         if (not exon_start) and (not exon_end):
             raise ValueError(msg)
 
         if exon_start:
-            if not values.get("element_genomic_start"):
-                msg = "Must give `element_genomic_start` if `exon_start` is given"
+            if not values.get("elementGenomicStart"):
+                msg = "Must give `elementGenomicStart` if `exonStart` is given"
                 raise ValueError(msg)
         else:
-            values["exon_start_offset"] = None
+            values["exonStartOffset"] = None
 
         if exon_end:
-            if not values.get("element_genomic_end"):
-                msg = "Must give `element_genomic_end` if `exon_end` is given"
+            if not values.get("elementGenomicEnd"):
+                msg = "Must give `elementGenomicEnd` if `exonEnd` is given"
                 raise ValueError(msg)
         else:
-            values["exon_end_offset"] = None
+            values["exonEndOffset"] = None
         return values
 
-    _get_transcript_val = field_validator("transcript")(return_value)
     model_config = ConfigDict(
+        # TODO: verify this example
         json_schema_extra={
             "example": {
                 "type": "TranscriptSegmentElement",
                 "transcript": "refseq:NM_152263.3",
-                "exon_start": 1,
-                "exon_start_offset": 0,
-                "exon_end": 8,
-                "exon_end_offset": 0,
-                "gene_descriptor": {
-                    "id": "normalize.gene:TPM3",
-                    "type": "GeneDescriptor",
+                "exonStart": 1,
+                "exonStartOffset": 0,
+                "exonEnd": 8,
+                "exonEndOffset": 0,
+                "gene": {
+                    "id": "hgnc:12012",
+                    "type": "Gene",
                     "label": "TPM3",
-                    "gene_id": "hgnc:12012",
                 },
-                "element_genomic_start": {
-                    "id": "fusor.location_descriptor:NC_000001.11",
-                    "type": "LocationDescriptor",
+                "elementGenomicStart": {
+                    "id": "NC_000001.11",
+                    "type": "SequenceLocation",
                     "label": "NC_000001.11",
-                    "location": {
-                        "type": "SequenceLocation",
-                        "sequence_id": "refseq:NC_000001.11",
-                        "interval": {
-                            "type": "SequenceInterval",
-                            "start": {"type": "Number", "value": 154192135},
-                            "end": {"type": "Number", "value": 154192136},
-                        },
-                    },
+                    "start": 154192135,
+                    "end": 154192136,
+                    # do we need a sequence reference here?
```

Comment:
yeah, it should probably have a SequenceReference, which points to the reference sequence upon which the location lies.  So should be something like

```
{
  "id": "ga4gh:SL.2K1vML0ofuYrYncrzzXUQOISRFJldZrO",  # the ID of the location on the reference sequence
  "type": "SequenceLocation",
  "sequenceReference": {
    "type": "SequenceReference",
    # easy way to get this -- type "refseq:NC_000001.11" into the sequence lookup util on fusion-builder and chop off the leading "ga4gh":
    "refgetAccession": "SQ.Ya6Rs7DHhDeg7YaOSg1EoNi3U_nQ9SvO"
  },
 "start": 154192135,
  "end": 154192135,
}
```

Reply:
the specific values don't matter obviously since we'll be regenerating this computationally later